common/src/main/java/com/wynntils/core/features/Feature.java

```diff
@@ -129,28 +124,13 @@ protected String getNameCamelCase() {
     protected void onInit(
             ImmutableList.Builder<Condition> conditions, ImmutableList.Builder<Class<? extends Model>> dependencies) {}
 
-    /**
-     * Called on enabling of Feature
-     *
-     * <p>Return false to cancel enabling, return true to continue. Note that if a feature's enable
-     * is cancelled it isn't called again by the conditions and must be done so manually, likely by
-     * the user.
-     */
-    protected boolean onEnable() {
-        return true;
-    }
-
-    /** Called on disabling of Feature */
-    protected void onDisable() {}
-
     /** Called to activate a feature */
     public final void enable() {
```

Comment:
The old code implied that it was a user responsibility to to check that you did not enable an already enabled feature. What is the rationale for changing this part of the interface?

Also, with the new state, if you call enable() on an unitialized feature, the result will be a no-op. Is this what you want?

Reply:
The rationale is that most of the time we were circumventing responsibility by using `tryEnable` or `tryDisable`, effectively making the code moot. Yes, that is intended

Reply:
Yes, but "most of the time" is not really an answer. My interpretation of this code is that we needed *some* way to get here *without* the checking done by `tryToEnable()`. So the interesting question is: who were calling `enable()` directly, without going through `tryToEnable`? It could be that it was a use case that has disappeared, so *all* calls go via tryToEnable. If that is the case, then your change is perfectly fine. 

But due to all the shuffling of logic, I cannot see now if there were some places where `enable` was called directly before, and what is happening with them. They will change behavior, and that might need discussion or argument about why the new behavior is correct.

Reply:
`enable` and `disable` are only called by `WynntilsCommand`'s `reload`, which not only is unaffected by `tryToEnable` vs `enable`, but in my opinion outdated.